src/main/java/com/example/businessmanagement2/repository/user/UserRepository.java

```diff
@@ -1,14 +1,30 @@
 package com.example.businessmanagement2.repository.user;
 
+import java.util.List;
+import java.util.Optional;
+import org.apache.ibatis.annotations.Delete;
+import org.apache.ibatis.annotations.Insert;
 import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
 import org.apache.ibatis.annotations.Select;
-
-import java.util.Optional;
+import org.apache.ibatis.annotations.Update;
 
 @Mapper
 public interface UserRepository {
 
-    @Select("SELECT id,company_name, user_name FROM users WHERE id = #{userId}")
-    Optional<UserEntity> findById(long userId);
+  @Select("SELECT id,company_name, user_name FROM users WHERE id = #{userId}")
```

Comment:
細かいですがSQLもなるだけ半角スペースを適切にいれて書きたいです。
```suggestion
  @Select("SELECT id, company_name, user_name FROM users WHERE id = #{userId}")
```

Reply:
半角スペース入れました。

a8f2878e403bacebd95445b9bf83a9dce9d853c3

Reply:
コミットログの貼り間違いですね。
直接Repository確認しました。
問題ないです。